django_pivot/pivot.py

```diff
@@ -36,9 +38,9 @@ def pivot(queryset, row, column, data, aggregation=Sum, choices='auto'):
     return queryset.values(*values).annotate(**annotations)
 
 
-def _get_annotations(column, column_values, data, aggregation):
+def _get_annotations(column, column_values, data, aggregation, prefix=''):
```

Comment:
I wonder if instead of a prefix, we should take any function that takes a string and returns a string. Prefixing seems like one specific use case only. If you want to do a prefix, you'd call the function like
```
pt = pivot(queryset, row, column, data, display_transform=lambda s: 'prefix_' + s)
```

And here we would define `_get_annotations` like:
```
def _get_annotations(column, column_values, data, aggregation, display_transform=lambda s: s):
    value = data if hasattr(data, 'resolve_expression') else F(data)
    return {
        display_transform(display_value): aggregation(Case(When(Q(**{column: column_value}), then=value)))
        for column_value, display_value in column_values
    }
```